adapters/rhomarkets/src/sdk/config.ts

```diff
@@ -0,0 +1,24 @@
+export const enum CHAINS {
+  SCROLL = 534352,
+}
+
+export const enum PROTOCOLS {
+  RHO_MARKETS = 1,
+}
+
+export const SUBGRAPH_URLS = {
+  [CHAINS.SCROLL]: {
+    [PROTOCOLS.RHO_MARKETS]: {
+      url: "https://api.studio.thegraph.com/query/79909/rho-scroll-staging/version/latest",
```

Comment:
Using this endpoint is a short term solution for the graph. There are rate limits on the test link from the subgraph studio. For production it will have to be deployed to the graph network and we will need a graph api key.

If you see what layerbank does, they are actually indexing using goldsky. This is probably easier tbh.

Reply:
> Using this endpoint is a short term solution for the graph. There are rate limits on the test link from the subgraph studio. For production it will have to be deployed to the graph network and we will need a graph api key.
> 
> If you see what layerbank does, they are actually indexing using goldsky. This is probably easier tbh.

I think you need update hourly_blocks.csv block.number to the latest one.Since we using a query method that precisely matches block numbers, if you need data from earlier blocks that include recent activities (where these accounts might not be active in the latest blocks), I will need to make the necessary adjustments.